src/tests.builds

```diff
@@ -22,7 +22,7 @@
     <SerializeProjects>true</SerializeProjects>
   </PropertyGroup>
 
-  <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
+  <Import Project="$(ToolsDir)tests.targets" Condition="Exists('$(ToolsDir)tests.targets')" />
```

Comment:
We checked, and it doesn't affect. Basically this is not considered a test project because it doesn't contain a `..\tests\..` folder in its path. That means that all targets are imported but none will run or be hooked.